.github/workflows/pre-release.yml

```diff
@@ -10,15 +10,20 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - target: x86_64-linux
+          - host: x86_64-linux
+            target: x86_64-linux
```

Comment:
This is definitely sth I wanted to do, I will keep it for https://github.com/pimalaya/core/issues/22.